code/script.js

```diff
@@ -0,0 +1,81 @@
+
+//DOM SELECTORS
+const username = 'HWallberg'
+const userbox = document.getElementById('userbox')
+const reposbox = document.getElementById('reposbox')
+
+//APIs
+const API_PROFIL = `https://api.github.com/users/${username}`
+const API_REPOS = `https://api.github.com/users/${username}/repos`
+
+
+//PICTURE
+const fetchProfilePicture = () => {
+    fetch(API_PROFIL)
+        .then(res => res.json())
+        .then(data => {
+            console.log(data)
+            picture = data.avatar_url
+
+    let profilePicture =    `<div class = "photobox">
+                                <p>${username}</p>
+                                <img class = "photo" src="${picture}" />
+                            </div>`;
+    return (userbox.innerHTML = profilePicture)
```

Comment:
Your code is clean and easy to follow everywhere, just this part seems a bit messy?